rbi/core/array.rbi

```diff
@@ -985,7 +985,7 @@ class Array < Object
   sig do
     params(
         arg0: Elem,
-        blk: T.proc.params().returns(Elem),
+        blk: T.proc.returns(Elem),
```

Comment:
This was triggering the new error about `params` requiring arguments, due to fixing the arguments in `GlobalState.cc`.